### PR TITLE
Datasets downloader

### PR DESCRIPTION
**Pull request by idchlife**
Hey! I created python script to download COCO datasets to make project startup easier and quicker.

I think this script may be tweaked/improved to include other datasets, but for now I suspect it will be good to have easy way to download COCO datasets with single script launch.

Script can be tested like this with small archive:

```bash
python downloader.py --test-run
```

### CODE REVIEW

**Comment by tucan9389** [comment]
@idchlife 
Thanks for your PR. I'll check ASAP. 👍

File: downloader.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  log("*** DOWNLOADING ***")
+  
+  dr.download()
+  
+log("")
+log("")
+log("*** SUCCESS ***")
+log("All datasets downloads complete!")
+log("*** SUCCESS ***")
+sys.exit(0)
```
Comment (tucan9389) [inline downloader.py]: @idchlife 
I think this line doesn't need because python call `exit(0)` when meet the end of the script.

**Review by tucan9389 — APPROVED** [review]
👍Thanks for your useful feature. 

> I added `psutil` on `requirements.txt`

File: downloader.py (hunk 1 above)
Comment (idchlife) [reply downloader.py]: @tucan9389 I agree! Removed this line

**Comment by idchlife** [comment]
@tucan9389 if you will need some kind of help with repository, not connected with machine learning, ping me, I'll do what I can! I can do python :)

I tried your another repository - https://github.com/tucan9389/PoseEstimation-CoreML and it was fascinating! Unfortunately, I understand that it was a temporary model there, so it was not very good with predicting poses in the "wild".
I guess the proper way to obtain good model is to train it yourself on your own system to some "good level".

Since I'm not very familiar with machine learning and how to train models properly for them to be good, I'm offerring any non machine learning help with code/readme in repository, so it will grow faster and help people like me learn how to train proper models in several easy commands in console and use them in CoreML/another format 👍 

Also, if you are interested in popularizing the pose prediction on mobile, I can help by creating repository and readme-manual to create pose prediction in Xamarin iOS project using repository mentioned above as base.

**Pull request merged**